### DISCUSSION TITLE
links in group descriptions or notes

### DISCUSSION THREAD

**Discussion by Oio-ief** [Ylianst/MeshCentral#4708] [Ideas]
Hello, how do I add a link to an email address or website to the group description or notes? Or add a device to a group by simply specifying a link without a redirect?

**Comment by si458** [comment]
The notes field is just plain text as far as I'm aware,  it's not html for links etc

**Reply by Oio-ief** [reply]
Each group is a customer and it is very convenient to add information in the form of links, for example, SIP Link

**Reply by Oio-ief** [reply]
Maybe there is another option for this?

**Reply by silversword411** [reply]
If those were markdown instead of text fields you could do simple formatting

**Reply by Oio-ief** [reply]
It would be great if there was a field in the group where you can insert links, contacts in the form of links.
